include/mantella_bits/orbitalMechanics.hpp

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+// C++ standard library
+#include <utility>
+#include <vector>
+
+// Armadillo
+#include <armadillo>
+
+namespace mant {
+  namespace itd {
+
+    // from pykep:
+    // ref_mjd2000(epoch(2451545.0,epoch::JD).mjd2000()
+    //inline double jd2mjd2000(const double & in){
+    //return ( in - 2451544.5 );
+    //}
+    // => 2451545.0 - 2451544.5
+    const double refModifiedJulianDay2000 = 0.5;
```

Comment:
I didn't understand what this is used for and why isn't this defined directly in line 251?